state-chain/chains/src/btc/api.rs

```diff
@@ -56,9 +58,12 @@ where
 			number_of_outputs: (btc_outputs.len() + 1) as u64, // +1 for the change output
 		})
 		.ok_or(AllBatchError::Other)?;
-
-		btc_outputs
-			.push(BitcoinOutput { amount: change_amount, script_pubkey: bitcoin_change_script });
+		if change_amount >= BITCOIN_DUST_LIMIT {
```

Comment:
we could write a function that adds the output, to save duplicating this condition, and it can just return the amount that was added (so we can just do `total_output_amount += add_btc_output(..,)` above